### PR TITLE
pa2 resummit

### PR DESCRIPTION
**Pull request by roberthuang0306**

### CODE REVIEW

**Comment by nianzelee** [comment]
@roberthuang0306 Please do not close a PR because we will lose all discussion.

**Review by nianzelee — CHANGES_REQUESTED** [review]
Please remove all out and diff files for PA2.

**Comment by roberthuang0306** [comment]
I have removed those files

**Comment by nianzelee** [comment]
> I have removed those files

You still change 107 files.
You move files from `lsv/pa1/` to `lsv/admin/lsv/pa2`. Why is this necessary?

**Comment by roberthuang0306** [comment]
可能不小心動到...我忘記了
請問助教那我要怎樣解決＠＠

**Comment by roberthuang0306** [comment]
助教：這樣應該好一點了
抱歉剛剛急了所以用了git push -f

File: 1bit-adder.blif [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.model fulladder
+.inputs a b
+.outputs cout
+
+.names a b cout
+-1 1
+.end
```
Comment (nianzelee) [inline 1bit-adder.blif:7]: Please delete this file.

File: fulladder.blif [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+10 1
+01 1
+.names k cin s
+10 1
+01 1
+.names a b cin cout
+11- 1
+1-1 1
+-11 1
+.end
```
Comment (nianzelee) [inline fulladder.blif:14]: Please delete this file.

File: lsv/pa2/master.csv [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+benchmarks/random_control/i2c.aig,Pass
+benchmarks/random_control/int2float.aig,Pass
+benchmarks/random_control/mem_ctrl.aig,Pass
+benchmarks/random_control/priority.aig,Pass
+benchmarks/random_control/router.aig,Pass
+benchmarks/arithmetic/adder.aig,Pass
+benchmarks/arithmetic/bar.aig,Pass
+benchmarks/arithmetic/max.aig,Pass
+benchmarks/arithmetic/sin.aig,Pass
+Score,91
```
Comment (nianzelee) [inline lsv/pa2/master.csv:15]: Please delete this file.

**Review by nianzelee — CHANGES_REQUESTED** [review]
Please do not change any file except your cpp file and report.

File: 1bit-adder.blif (hunk 1 above)
Comment (roberthuang0306) [reply 1bit-adder.blif]: I have removed it~

File: fulladder.blif (hunk 2 above)
Comment (roberthuang0306) [reply fulladder.blif]: I have removed it~

File: lsv/pa2/master.csv (hunk 3 above)
Comment (roberthuang0306) [reply lsv/pa2/master.csv]: I have removed it~

**Comment by nianzelee** [comment]
There are still several strange files. Please have a look at "Files changed".

**Comment by nianzelee** [comment]
Perfect! Your Git skills are improved!

**Comment by roberthuang0306** [comment]
yeah!! 謝謝助教～～我覺得以後還是不要用`git add .`好了...

**Comment by nianzelee** [comment]
> yeah!! 謝謝助教～～我覺得以後還是不要用`git add .`好了...

Exactly. You must be very precise about which files you want to operate. It is the spirit of version control.
I hope you will benefit from what you learned in the PAs.

**Pull request merged**